### DISCUSSION TITLE
Installation guide

### DISCUSSION THREAD

**Discussion by matthiserver** [mhaeuser/Battery-Toolkit#17] [General]
Hi,
can somebody please explain me, what I exactly have to do to get the app to work on my Mac. Because I don't get any dmg-file or something like that, just all the project files and bins.
Thanks very much in advance.

**Comment by mhaeuser** [comment]
You download releases here (and just move the app to /Applications): https://github.com/mhaeuser/Battery-Toolkit/releases

Many people seem to not find the Releases page (and GitHub somehow tries hard to not make it too visible), so I might extend the README next release.

**Reply by matthiserver** [reply]
thank you very much. It would be very nice if you extend the README, because I was looking there for how to install it. 
I have one other question: What's the difference between dSYMs and the zip-file without?

**Reply by mhaeuser** [reply]
dSYMs are for debugging, they are irrelevant for normal usage.

**Reply by codeskipper** [reply]
then I suggest leaving the one that includes dSYMs out of the releases as it is bound to confuse most people :-)

**Reply by mhaeuser** [reply]
Well, maybe things changed as I tried to polish the app a lot, but I still consider it a hack. There’s no auto-update, there’s no Notarization. I expect more technically inclined people to use it - somehow. I will definitely not drop dSYM, because why wouldn’t I give people the option to properly debug the app, but I will probably expand the README - some day.

**Comment by matthiserver** [comment]
When I now want to open the .app-File, it says: The app cannot be opened because Apple cannot check it for malware. What do I have to do and why can't Apple check it (because I've already downloaded other applications from github for MaC (e.g. Stats) and there this problem didn't occur.
Thanks in advance

**Reply by mhaeuser** [reply]
Battery Toolkit is signed with a development certificate, because I don’t have a Developer Program membership. You can either open it from the context menu or use the Terminal to remove the quarantine flag.

**Reply by kapad** [reply]
I don't see any option for running this app in the context menu. Is there something that I am missing? running macOS Sequoia 15.1.1

**Reply by mhaeuser** [reply]
What context menu?

**Reply by kapad** [reply]
The one that you mentioned in your previous comment. 

> Battery Toolkit is signed with a development certificate, because I don’t have a Developer Program membership. You can either open it from the context menu or use the Terminal to remove the quarantine flag.

**Reply by kapad** [reply]
but I found the issue. It's to do with macOS Sequoia. And I had to go to settings->security and then allow the app from there.

**Reply by mhaeuser** [reply]
> The one that you mentioned in your previous comment.

Ugh, that’s what happens when you respond half asleep… Sorry, yes. :) Maybe I should expand the README some day…
